api/HTMLAnchorElement.json

```diff
@@ -461,12 +461,10 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": "81"
```

Comment:
This change seems wrong. (I just verified in Chrome 81 that HTMLAnchorElement has a `hrefTranslate` property.)